module1.js

```diff
@@ -16,7 +16,11 @@
 function question1(name1) {
   if (name1) {
     // TODO:
-    return;
+    var str = name1;
+    var res = str.match(/Code Your Future/g);
+    return true;
```

Comment:
Note that you're not actually _using_ `res`, which is a bad sign; your current implementation could be simplified to `return !!name1;`. Also the assignment `var str = name1` is redundant, you can refer to `name1` directly. Finally a regular expression is overcomplicated and actually _breaks_ the required functionality; what if the input was `'A string that mentions Code Your Future but is not right'`?